doc/userguide/rules/noalert.rst

```diff
@@ -0,0 +1,37 @@
+Alert Keywords
+==============
+
+In addition to the action, alerting behavior can be controlled in the rule body using the ``noalert`` and ``alert`` keywords.
+Additionally, alerting behavior is controlled by :doc:`thresholding`.
+
+noalert
+-------
+
+A rule that specifies ``noalert`` will not generate an alert when it matches, but rule actions will still be performed.
+
+``noalert`` is often used in rules that set a ``flowbit`` for common patterns.
+
+``noalert`` is meant for use with rule actions ``alert``, ``drop``, ``reject`` that all explicitly or implicitly include ``alert``.
+
+.. container:: example-rule
+
+   alert http any any -> any any (http.user_agent; content:"Mozilla/5.0"; startwith; endswith; \
+   flowbits:set,mozilla-ua; :example-rule-options:`noalert;` sid:1;)
```

Comment:
I have no idea why doesn't it fail the CI but locally, a `make html` would fail due to the following patch missing from this file:
```diff
diff --git a/doc/userguide/rules/noalert.rst b/doc/userguide/rules/noalert.rst
index c27d03b34..51dfe5de1 100644
--- a/doc/userguide/rules/noalert.rst
+++ b/doc/userguide/rules/noalert.rst
@@ -1,6 +1,8 @@
 Alert Keywords
 ==============
 
+.. role:: example-rule-options
+
 In addition to the action, alerting behavior can be controlled in the rule body using the ``noalert`` and ``alert`` keywords.
 Additionally, alerting behavior is controlled by :doc:`thresholding`.
```